axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -623,34 +623,34 @@
       "finality": "finalized",
       "approxFinalityWaitTime": 1
     },
-    "stellar-test2": {
+    "stellar-test-2025-q1": {
       "name": "Stellar",
-      "axelarId": "stellar-test2",
+      "axelarId": "stellar-test-2025-q1",
       "rpc": "https://soroban-testnet.stellar.org",
       "horizonRpc": "https://horizon-testnet.stellar.org",
       "networkType": "testnet",
-      "chainType": "stellar",
```

Comment:
why remove this?

Reply:
Good catch. Not sure what that field does. will added it back.